azure-iot-device/azure/iot/device/abomination.py

```diff
@@ -0,0 +1,227 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+"""This module provides hard coded patches used to modify items from the libraries.
+Currently we have to do like this so that we don't use exec anywhere"""
+
+
+def execute_atrocity_for_sync():
```

Comment:
Where are the processes for using this documented? Before, patch.py documented what it did, but now that we're using it just to generate output, which we then use here, when and how does this get updated?

Is this (and the other abomination) manually maintained? Or do we regenerate from patch.py? And if we do generate from patch.py, could we add a script or something for doing so?

Reply:
i feel we should keep patch around coz we need the output of patch to append onto these 2 files

---
In reply to: [389206044](https://github.com/Azure/azure-iot-sdk-python/pull/482#discussion_r389206044) [](ancestors = 389206044)